app/assets/stylesheets/v3/layout_components/forms.scss

```diff
@@ -178,14 +179,15 @@ input[type="checkbox"] + label,
     content: " ";
     display: block;
     position: absolute;
-    left: 10px;
-    top: calc(50% - 12px);
-    width: 24px;
-    min-width: 24px;
-    height: 24px;
+    left: 1rem;
+    top: calc(50% - 15px);
+    width: 30px;
+    min-width: 30px;
+    height: 30px;
     background-color: $white;
-    border: $base-border;
+    border: $form-border;
     transition: border-color 0.25s linear;
+    border-radius: 6px;
```

Comment:
should this be 9px? using $base-radius?

Reply:
I tried 9px, but because the checkbox is fairly small it made it look too circular.